AlphaWallet/Activities/ViewModels/ActivityViewModel.swift

```diff
@@ -162,8 +163,12 @@ extension HasTokenImage {
 
     public func icon(withSize size: GoogleContentSize) -> Subscribable<TokenImage> {
         let name = symbol.nilIfEmpty ?? name
-        let colors = [R.color.radical()!, R.color.cerulean()!, R.color.emerald()!, R.color.indigo()!, R.color.azure()!, R.color.pumpkin()!]
+        let colors = Constants.defaultTokenIconColors
```

Comment:
Can you help separate this one out (and if there are others), into a separate commit at least?